kola/tests/misc/nfs.go

```diff
@@ -110,9 +118,9 @@ systemd:
 	c.Log("NFS client booted.")
 
 	checkmount := func() error {
-		status, err := c.SSH(m2, "systemctl is-active mnt.mount")
+		status, err := c.SSH(m2, "systemctl is-active var-mnt.mount")
```

Comment:
Shouldn't this use `mountLocation`?

Reply:
`mountLocation` might be a mis-leading variable name. It's intended to be what you're mounting on the remote system (in the service the `What` for the mount is defined as `<PrivateIP>:<mountLocation>`, the `Where` is always `/var/mnt`). No matter it's value the systemd mount will be `var-mnt.mount`.